index.html

```diff
@@ -86,9 +89,22 @@ <h2><a href="./layouts/">Layouts</a></h2>
     </div>
```

Comment:
lets remove this footer - years are irrelevant and the company isn't in san francisco anymore